modules/oneVideoBidAdapter.md

```diff
@@ -12,41 +12,40 @@ Connects to One Video demand source to fetch bids.
 # Test Parameters
 ```
     var adUnits = [
-	{
-      code: 'video1',
-        sizes: [640,480],
-        mediaTypes: {
-          video: {
-            context: "instream"
-          }
-        },
-        bids: [
-          {
-            bidder: 'oneVideo',
-            params: {
-              video: {
-                playerWidth: 480,
-                playerHeight: 640,
-                mimes: ['video/mp4', 'application/javascript'],
-                protocols: [2,5],
-                api: [2],
-                position: 1,
-                delivery: [2],
-                playbackmethod: [1,5],
-                placement: 123,
-                sid: <scpid>,
-                rewarded: 1
+        {
+            code: 'video1',
+              sizes: [640,480],
+              mediaTypes: {
+                video: {
+                  context: "instream"
+                }
               },
-             },
-              site: {
-                id: 1,
-                page: 'http://abhi12345.com',
-                referrer: 'http://abhi12345.com'
-              },
-              pubId: 'brxd'
-            }
-          }
-        ]
-    }
-	];
+              bids: [
+                {
+                  bidder: 'oneVideo',
+                  params: {
+                    video: {
+                      playerWidth: 480,
+                      playerHeight: 640,
+                      mimes: ['video/mp4', 'application/javascript'],
+                      protocols: [2,5],
+                      api: [1],
+                      position: 1,
+                      delivery: [2],
+                      playbackmethod: [1,5],
+                      sid: <scpid>,
+                      rewarded: 1,
+                      display: 1
```

Comment:
Should this be a `1` still?  Per the current adapter code, this would tell the adapter it should be a `banner` type ad rather than a `video` ad.

Reply:
@jsnellbaker yes the display should be 1 but we are not updating in .md file because this is not for all publishers(video) and we are pushing this code for testing with one banner publisher so we will communicate to only that publisher as of now. we are not expecting from other publishers display:1

Reply:
@jsnellbaker updated the markup in modules/oneVideoBidAdapter.md  review pls